Week-3/Exercise-3.1/script.js

```diff
@@ -0,0 +1,22 @@
+//Given reducer method:
+function add(a, b) {
```

Comment:
Did the exercise not say to support n number of params?

Reply:
resolved and committed